react/components/SelectionListOrderBy.js

```diff
@@ -28,13 +28,14 @@ const SelectionListOrderBy = ({ intl, orderBy, options }) => {
     hints: { mobile },
   } = useRuntime()
 
-  const renderOptions = () => {
+  const renderOptions = orderBy => {
     return options.map(option => {
       return (
         <SelectionListItem
           key={option.value}
           onItemClick={handleOutsideClick}
           option={option}
+          selected={option.value == orderBy}
```

Comment:
:pick: (nitpick)
```suggestion
          selected={option.value === orderBy}
```

Reply:
`===` is strict comparison, which is generally safer. `==` converts types (e.g. `1 == '1'` returns true) which might introduce bugs